packages/main/src/ListItem.ts

```diff
@@ -254,6 +254,10 @@ abstract class ListItem extends ListItemBase {
 	}
 
 	async _onkeydown(e: KeyboardEvent) {
+		if ((isSpace(e) || isEnter(e)) && this.onSpacePressForInputs(e)) {
```

Comment:
consider placing this check (this.onSpacePressForInputs(e)) in the below if block (on line 266) as third condition.

Reply:
I tried placing the check in the suggested if block, but it doesn’t work as expected that way